camayoc/api.py

```diff
@@ -131,6 +131,11 @@ def get(self, endpoint, **kwargs):
         url = urljoin(self.url, endpoint)
         return self.request('GET', url, **kwargs)
 
+    def options(self, endpoint, **kwargs):
+        """Send an HTTP OPTIONS request."""
+        url = urljoin(self.url, endpoint)
+        return self.request('OPTIONS', url, **kwargs)
+
```

Comment:
This I added more for my convenience while exploring the API in a jupyter notebook than for use in tests, though we could test that it provides correct methods if you thought that was something is not just automatically generated by Django

Reply:
Should the default headers be called here?

Reply:
I need to rebase on master and do that